### PR TITLE
Add back button to profile from incident report view

### PR DESCRIPTION
**Pull request by kushaljh**

### CODE REVIEW

File: src/ResidentDirectory.js [hunk 1]
```diff
@@ -26,7 +26,7 @@ const colorStylesResident = {
             backgroundColor: isFocused ? '#B2D4FF' : 'white',
             color: (data.inResidence != null && !data.inResidence) ? 'red' : 'black',
             ":active": {
-                backgroundColor: "#B2D4FF"
+                backgroundColor: "white"//"#B2D4FF"
```
Comment (kushaljh) [inline src/ResidentDirectory.js]: Does this not mess with the highlighting?

File: src/ResidentDirectory.js [hunk 2]
```diff
@@ -37,6 +37,7 @@ class ResidentDirectory extends Component {
         super(props);
         this.state = {
             startDate: '?',
+            cardHeader: "Select...",
```
Comment (kushaljh) [inline src/ResidentDirectory.js]: This way you have created a fake header I believe 
Right?

File: src/ResidentDirectory.js [hunk 3]
```diff
@@ -236,8 +248,8 @@ class ResidentDirectory extends Component {
                         <Card style={{width: '30rem', marginTop: "30px"}}>
                             <Card.Body>
                                 <Card.Title>
-                                    <Card.Img variant="top" src="person.png" style={{height: 30, width: 30}}/> 
-                                    {this.state.resident.FIRST_NAME + " " + this.state.resident.LAST_NAME}
+                                    <Card.Img variant="top" src="person.png" style={{height: 30, width: 30}}/>
+                                    {this.state.cardHeader}
```
Comment (kushaljh) [inline src/ResidentDirectory.js]: The dropdown selection and card header should be two separate variables so that we know the difference

**Review by wilsmic2 — APPROVED** [review]

File: src/ResidentDirectory.js (hunk 1 above)
Comment (aashnasheth) [reply src/ResidentDirectory.js]: I was playing around and forgot to revert -- will switch it back to #B2D4FF

**Review by aashnasheth — APPROVED** [review]

**Pull request merged**